### PR TITLE
Add Macedonian translation

### PR DESCRIPTION
**Pull request by violeta-p**
Hello,

My name is Violeta and I added Macedonian (mk) translation for TryRuby.
Any suggestions for improvement are welcomed.

Thank you!

### CODE REVIEW

**Comment by hsbt** [comment]
Thanks, But I'm not sure this pull-request is the correct translation.

Can you try the cross review with Macedonian people?

**Comment by violeta-p** [comment]
@hsbt,

Can you please recommend a proper way for such review?

File: translations/mk/try_ruby_10.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+lang:   MK
+title:  Имаш 30 минути? Започни со програмирање во Руби веднаш!
+answer: 
+class:  stretcher chapmark
+ok:     
+error:  
+---
+
+Руби (анг. **Ruby**) е програмски јазик од Јапонија кој го револуционизира софтверскиот развој.
```
Comment (Mar1anne) [inline translations/mk/try_ruby_10.md]: I would actually not translate the name. I also checked the other translations, they use Ruby as well. 

> Ruby е програмски јазик од Јапонија кој го револуционизира софтверскиот развој.

File: translations/mk/try_ruby_110.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title:  Јаболката се различни од портокалите
+answer: \[\]
+ok:     Одлично, ова е празна листа
+error:
+---
+
+И броевите се различни од стринговите.
+
+Иако можеш да користиш методи врз било кој објект во Руби, сепак некои методи единствено функционираат само за одредени типови.
+Но, секогаш можеш да извршиш конверзија помеѓу разни типови користејќи ги Рубиевите **to** методи.
```
Comment (Mar1anne) [inline translations/mk/try_ruby_110.md]: Maybe a better format would be: 

> користејќи ги "**to**" методите во Ruby. 

or simply

> користејќи ги "**to**" методите.

File: translations/mk/try_ruby_160.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+lang:   MK
+title:  Зачувано и складирано
+answer: \[12, 35, 47\]
+load:   ticket = [12, 47, 35]
+ok:     Ти ја подреди листата
+error:  
+---
+
+Сѐ до сега се придржуваше до твоите лото броеви, ставајќи ги во променлива, односно __варијабла__ (анг. variable), наречена ticket.
```
Comment (Mar1anne) [inline translations/mk/try_ruby_160.md]: Maybe use **променлива** only. No need to use it in brackets later on.

> варијабли (променливи)

File: translations/mk/try_ruby_540.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+error:
+load:   prev;class Blurbalizer;attr_reader :title;def initialize(title);@title=title;@blurbs=[];end;def add_a_blurb(mood, content);@blurbs << Blurb.new(mood, content);@blurbs.each {|t| t.time -= 73};end;def show_timeline;puts "Blurbalizer: #{@title} has #{@blurbs.count} Blurbs\n";@blurbs.sort_by { |t| t.time}.reverse.each { |t| puts "#{t.content.ljust(40)} #{t.time}"};end;end;myapp = Blurbalizer.new "The Big Blurb";myapp.add_a_blurb :sick,"Today Mount Hood Was Stolen!";myapp.add_a_blurb :confused,"I can not believe Mt. Hood was stolen!";myapp.add_a_blurb :dazed,"I am speechless!";myapp.add_a_blurb :mad,"It was stolen by a giraffe !!";myapp.add_a_blurb :sad,"I Left my Hoodie on the Mountain!";myapp.add_a_blurb :mad,"I am never going back to that mountain."
+---
+
+Одлично, пристина до уште една нова лекција! Дали сфати што прави кодот во  Blurbalizer<sup>TM</sup> класата?
+
+> Во Blurbalizer<sup>TM</sup> има два __методи__ (add\_a\_blurb and show\_timeline).
+> Ти можеш да користиш метод надвор од класата, на ист начин како што направивме со accessors.
+
+Време е да почнеме да ја користиме Blurbalizer<sup>TM</sup>. Јас веќе вчивав (анг. loaded) неколку
```
Comment (Mar1anne) [inline translations/mk/try_ruby_540.md]: *вчитав

File: translations/mk/try_ruby_550.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+Некои убави работи можат да бидат направени со едноставни делови од Руби, посебно кога ќе ги
+искомбинираш заедно за да креираш нови работи.
+
+Тука, ние имаме апликација направена од класа која содржи друга класа. И, всушност, Руби навистина добро
+функционира со ваквите вгнездувања. Тоа се нарекува објектно ориентирано програмирање.
+
+Ние стигнавме до __последната вежба за програмирање__ од TryRuby. Доколку сакаш, можеш да додадеш
+и други перформанси (анг. features) на Blurbalizer<sup>TM</sup>.
```
Comment (Mar1anne) [inline translations/mk/try_ruby_550.md]: features -> карактеристики / особини

File: translations/mk/try_ruby_60.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+lang:   MK
+title:  Кажи ни го твоето име во обратен редослед
+answer: \w+
+ok:     репуС
+error:  
+---
+
+Стрингот (анг. **String**) е сет од карактери кои компјутерот може да ги процесира. Замисли дека буквите се обесени на жица за алишта и наводниците се штипалки кои ги држат краевите. Наводниците го означуваат почетокот и крајот.
```
Comment (Mar1anne) [inline translations/mk/try_ruby_60.md:9]: I would use **String** as is, not translated.

**Comment by Mar1anne** [comment]
@violeta-p @hsbt 

Hello :) 

I reviewed the translation, and aside from a couple of minor fixes, the rest looks excellent. 
I also made a couple of suggestions, but they are not necessary.  

@violeta-p Thank you for translating TryRuby to Macedonian 😍

File: translations/mk/try_ruby_60.md (hunk 6 above)
Comment (violeta-p) [reply translations/mk/try_ruby_60.md:9]: For this one Mariana I'd stick to **Стринг**. From the resources I found online, the word in Cyrillic is what is used in official Macedonian translations, but of course I am open to reviewing this deeper.

**Comment by violeta-p** [comment]
@Mar1anne,
I'd like to thank you enormously for taking the time and energy to thoroughly review the translation! ❤️ 


@hsbt,
I implemented Mariana's suggestions to improve the translation.
Please let me know if this review is enough or we should ask for more opinions.


Thank you in advance,
Violeta

**Comment by hsbt** [comment]
@violeta-p @Mar1anne Great! Thanks to contributions.

**Pull request merged**